.NET/Microsoft.Recognizers.Text.DataDrivenTests/NumberWithUnit/TestNumberWithUnit_Japanese.cs

```diff
@@ -6,6 +6,7 @@ namespace Microsoft.Recognizers.Text.NumberWithUnit.Tests
     [TestClass]
     public class TestNumberWithUnit_Japanese : TestBase
     {
+        /*
```

Comment:
Same here and this one is even more critical. We can't disable multiple existing types already in use.